### PR TITLE
Enable vet shadow

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

**Comment by favadi** [comment]
[Avoid using shadowing variable](https://trello.com/c/7BcSvJ4f/66-avoid-using-shadowing-variable)

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ FROM golang
 ADD . /go/src/github.com/KyberNetwork/reserve-data
 
 ENV KYBER_EXCHANGES huobi,bittrex,binance
-WORKDIR /go/src/github.com/KyberNetwork/reserve-data
```
Comment (favadi) [inline Dockerfile]: that could break production installation. What is the reason of this change?

File: blockchain/gas_oracle.go [hunk 2]
```diff
@@ -42,8 +42,8 @@ func (self *GasOracle) RunGasPricing() error {
 		return err
 	}
 	defer func() {
-		if err := r.Body.Close(); err != nil {
-			log.Panic(err)
+		if vErr := r.Body.Close(); vErr != nil {
```
Comment (favadi) [inline blockchain/gas_oracle.go]: cErr.

File: Dockerfile (hunk 1 above)
Comment (halink0803) [reply Dockerfile]: If I change the config.json file to use relative path for keystore file. I have to change the Dockerfile as so to be able to run. If I don't change docker run break.
